libs/types/src/dtos/common/params.dto.ts

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line max-classes-per-file
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsNumberString, IsUrl } from 'class-validator';
+
+export class MsaIdParam {
+  @ApiProperty({ type: String, description: 'MSA Id of account' })
+  @IsNumberString({ no_symbols: true })
```

Comment:
I'd actually be in favor of creating a custom validation decorator, that can check not only for a valid numeric integer string, but also validate that it's > 0 (or any other simple validations on an MSA that don't rely on the chain)

Reply:
I agree. I'll create customer MsaId and schemaId validators in here. 
On the side note `no_symbol` forces the numbers to be > 0